packages/web/src/common/store/pages/signon/selectors.ts

```diff
@@ -12,9 +12,10 @@ export const getNameField = (state: AppState) => state.signOn.name
 export const getPasswordField = (state: AppState) => state.signOn.password
 export const getHandleField = (state: AppState) => state.signOn.handle
 export const getIsVerified = (state: AppState) => state.signOn.verified
-export const getCoverPhotoField = (state: AppState) => state.signOn.coverPhoto
+export const getCoverPhotoField = (state: AppState) =>
+  state.signOn.coverPhoto?.value
 export const getProfileImageField = (state: AppState) =>
-  state.signOn.profileImage
+  state.signOn.profileImage?.value
```

Comment:
I feel like these should behave the same as the other ones. It would be weird to need to do `x.value` for some but not for these

Reply:
Fair. Updated them 👌